src/Admin/AbstractAdmin.php

```diff
@@ -240,7 +240,7 @@
      * The related parent association, ie if OrderElement has a parent property named order,
      * then the $parentAssociationMapping must be a string named `order`.
```

Comment:
This comment is not right anymore, it could be also an array. it will only be an string for BC purposes